engine/src/main/java/org/terasology/rendering/dag/stateChanges/ReflectedCamera.java

```diff
@@ -47,7 +46,7 @@ private ReflectedCamera(Camera camera, boolean reflected) {
         this.camera = camera;
         this.reflected = reflected;
 
-        if (reflected == false) {
+        if (!reflected) {
```

Comment:
Was this also triggering a warning?

Reply:
This triggers the [SimplifyBooleanExpression] warning.

Reply:
Thank you for letting me know. I hadn't realized the !-based form is the favored one. I'll try and use it more often.